common/views/components/Tags/Tags.js

```diff
@@ -18,8 +18,8 @@ function buildTag(tag) {
 const Tags = ({ tags }: Props) => (
   <div className="tags">
     <ul className="tags__list plain-list no-margin no-padding flex flex--wrap">
-      {tags.map(tag => (
-        <li key={tag.name} className={`tags__tag ${font({s: 'HNL5', m: 'WB7'})} ${spacing({s: 2}, {margin: ['right', 'bottom']})}`}
+      {tags.map((tag, i) => (
+        <li key={`tag${i}`} className={`tags__tag ${font({s: 'HNL5', m: 'WB7'})} ${spacing({s: 2}, {margin: ['right', 'bottom']})}`}
```

Comment:
I know it doesn't matter here (because we're not going to be reactively updating these tags after initial render), but we shouldn't use indexes as keys – I think `tag.text` or `tag.url` would be better for the key here, since it's going to be unique?

Reply:
But as this is just for Nunjucks in order to stop getting errors/warnings, this is probably cool.

Reply:
For my info:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318